search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,307 @@
+
+### Exact match and patterns
+
+TMDQ can support exact match on all properties (subject to database collation) regardless of the property index configuration in the data model. All text properties can support pattern matching. The database index supports a fixed number of leading characters. The database store a maximum string size before it overflows to another form. Only short form strings can be used in database queries.
+
+Elasticsearch supports exact match on all non-text properties. Text properties only support exact and pattern matches if set to tokenised `both` or `false` in the data model. Elasticsearch provides supports values up to approximately 32,700 UTF-8 bytes.
+
+The following specific CMIS QL fields are supported:
+
+* `cmis:parentId`
+* `cmis:objectId`
+* `cmis:objectTypeId`
+* `cmis:baseTypeId`
+* `cmis:contentStreamMimeType`
+* `cmis:contentStreamLength`
+
+The following CMIS QL comparison operators are supported:
+
+* `=`, `!=,` `<>`, `<`, `<=`, `>`, `>=`
+* `IN`, `NOT IN`, `LIKE`
+
+The following AFTS exact matches and patterns are supported:
+
+* `=<field>:term`
+* `=<field>:ter*`
+* `=<field>:*erm`
+
+### Support for special fields in TMDQ using AFTS
+
+* PARENT
+* TYPE
+* ASPECT
+* EXACTTYPE
+* EXACTASPECT
+
+> **Note:** CMIS QL does not support any use of CONTAINS() using the database.
+
+Transactional Metadata Query and the Elasticsearch index queries are intended to support different use cases. They differ in queries and options that they support and in the results they generate with respect to collation and scoring.
+
+## Transactional metadata queries supported by database
+
+Use this information to understand the queries supported by the database.
+
+The Alfresco Full Text Search (FTS) query text can be used standalone or it can be embedded in CMIS-SQL using the `contains()` predicate function. The CMIS specification supports a subset of Alfresco FTS. For more information on search syntax, see [Alfresco Full Text Search Reference]({% link search-services/latest/using/index.md %}).
+
+**CMIS QL**
+
+The following object types and their sub-types are supported:
+
+* `cmis:document`
+
+    For example:
+
+    ```sql
+    select * from cmis:document
+    ```
+
+* `cmis:folder`
+
+    For example:
+
+    ```sql
+    select * from cmis:folder 
+    ```
+
+* Aspects
+
+    For example:
+
+    ```sql
+    select * from cm:dublincore 
+    ```
+
+## CMIS property data types
+
+The `WHERE` and `ORDER BY` clauses support the following property data types and comparisons:
+
+* `string`
+  * Supports comparisons using `=`, `<>`, `<`, `<=`, `>=`, `>`, `IN` and `NOT IN` `LIKE`
+  * Supports ordering for single-valued properties
+    For example:
```

Comment:
This renders on the same line as the previous sentence giving a capital letter in the middle of a sentence.

Reply:
Right I see, I have adjust how it is coded.